pkg/prompts/prompts.go

```diff
@@ -155,6 +165,25 @@ func validateBiggerThanZero(input string) error {
 	return nil
 }
 
+func validateSemanticVersion(input string) error {
+	_, err := version.Parse(input)
+	return err
+}
+
+func (r *realPrompter) CaptureSemanticVersion(promptStr string) (*version.Semantic, error) {
```

Comment:
Is this different than `CaptureVersion`?

Reply:
No! Missed that, will remove.

Reply:
Actually, my version returned an actual `*version.Semantic` object instead of a string. Removed nonetheless.